packages/editor/package.json

```diff
@@ -34,6 +34,7 @@
 	],
 	"dependencies": {
 		"@babel/runtime": "7.25.7",
+		"@floating-ui/react-dom": "2.0.8",
```

Comment:
Why can't be use the popover component again?

Reply:
It’s been a while since I worked on this PR. I’ll need to review it again and will get back to you.